CMakeLists.txt

```
@@ -14,7 +14,7 @@ set(IGN_CMAKE_VER ${ignition-cmake2_VERSION_MAJOR})
 #============================================================================
 # Configure the project
 #============================================================================
-ign_configure_project(VERSION_SUFFIX pre2)
+ign_configure_project(VERSION_SUFFIX)
```

Leaving a VERSION_SUFFIX empty looks weird. I assume it would work and it is convenient for the next time. 
```suggestion
ign_configure_project()
```

I would remove it but just a personal taste.

---

Ahh I've been leaving it everywhere :confused: I'm inclined to leaving it like this now just so all libraries follow the same pattern. But I agree it looks weird.